scripts/build_devimage.sh

```diff
@@ -19,6 +19,7 @@ set -euo pipefail
 CONTEXT_DIR=devbuild
 IMAGE_NAME=kserve/modelmesh-controller-develop
 DEV_DEPS="$0 Dockerfile.develop go.mod go.sum .pre-commit-config.yaml"
+ENGINE=$1
```

Comment:
may as well keep default behaviour here too
```suggestion
ENGINE=${1:-docker}
```